### PR TITLE
[CLI] Support schedule param to be passed as bool

### PR DESCRIPTION
**Pull request by GiladShapira94**
Added an option to support schedule equal true within the mlrun CLI.

The user will need to set a string value for example: `--schedule "true"`.

### CODE REVIEW

**Review by quaark — CHANGES_REQUESTED** [review]
A couple minor comments

File: mlrun/projects/pipelines.py [hunk 1]
```diff
@@ -720,6 +720,7 @@ def run(
                 image=mlrun.mlconf.default_base_image,
             )
             msg = "executing workflow "
+            print(" workflow_spec.schedule_2:", workflow_spec.schedule)
```
[the remark this answers is not in the document]
Comment (GiladShapira94) [reply mlrun/projects/pipelines.py]: Fixed

File: mlrun/__main__.py [hunk 2]
```diff
@@ -896,7 +896,8 @@ def logs(uid, project, offset, db, watch):
     "--schedule",
     type=str,
     default=None,
-    help="a standard crontab expression string, for help see: "
+    help="a standard crontab expression string, for using the pre-defined workflow's schedule, set --schedule true. "
+    "for help see: "
```
Comment (Tankilevitch) [inline mlrun/__main__.py:900]:
```suggestion
    help="to create a schedule define a standard crontab expression string, to use the pre-defined workflow's schedule, set --schedule true."
    "for help see: "
```

**Review by Tankilevitch — APPROVED** [review]
minor suggestion

**Pull request merged**